crowdin.conf

```diff
@@ -1,7 +1,7 @@
 [crowdin.project]
 projectid = "design-system"
 repo.branch = "master"
-source.base = "."
+source.base = "./i18n"
```

Comment:
Can just be `i18n`, it will be made relative to the crowdin.conf file location anyway.